.changelog/1120.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+add vault_secrets_integration_confluent resource
+```
```

Comment:
We should mention that we are also adding the `hcp_vault_secrets_rotating_secret` resource

Reply:
hcp_vault_secrets_rotating_secret was already there we are just modifying it to support confluent secret